public/assets/javascript/backend.js

```diff
@@ -6422,38 +6422,6 @@ if(!Object.keys)
     };
 }
 
-// =================================== START DATEPICKER =====================================
```

Comment:
all cleanup here, public/assets/stylesheet/application.css and app/Http/Controllers/OrganiserCustomizeController.php should be in a separated commit / PR, what do you think ?